CHANGELOG

```diff
@@ -1,4 +1,16 @@
 # Changelog
+## [0.2.8] - 2020-03-10
+### Bug Fixes
+ - Fix adding slave to newly create bridge causes bridge link down.
+ - Fix cannot switch bond mode from 1(active-backup) to 5(balance-tlb)
+
+### Known Issues with NetworkManager Version 1.22.8
+ - All changes to bridges might disrupt their connectivity.
```

Comment:
IMHO it would be nicer to add the links to Bugzillas/issues to be able to easier follow up on this.

Reply:
I think that is going to add too much noise to the CHANGELOG. What do you think? @cathay4t @EdDev

Reply:
Adding a link to the BZ/Issue sounds good to me.

I would also consider having a section if this `Known Issues` that is moved rather than copied all the time.

Reply:
We can evaluate that. Could you propose a format for that? @EdDev

Reply:
> I think that is going to add too much noise to the CHANGELOG. What do you think? @cathay4t @EdDev

FYI, bugzilla supports shorter URLs like https://bugzilla.redhat.com/1806969 which can help to reduce the noise.

Reply:
I've added the links. Thanks!